### PR TITLE
no else return

### PR DESCRIPTION
**Pull request by Korilakkuma**
### This PR will ...

This PR will unify "no else return".

### Why is this Pull Request needed ?

- Unify code format
- Low nest level

### CODE REVIEW

**Comment by kqyang** [comment]
@Korilakkuma Thanks for the pull request, but can you please explain the background of this change? What problem is this CL trying to address?

**Comment by Korilakkuma** [comment]
@kqyang Thanks for comment.

> the background of this change

Because it seems using a early return, I fixed code that does not using a early return.

> What problem is this CL trying to address?

This PR will unify coding style and improve readability by low nest level..

File: packager/hls/base/simple_hls_notifier.cc [hunk 1]
```diff
@@ -380,9 +381,8 @@ bool SimpleHlsNotifier::NotifyNewSegment(uint32_t stream_id,
         if (!WriteMediaPlaylist(master_playlist_dir_, playlist))
           return false;
       }
-    } else {
-      if (!WriteMediaPlaylist(master_playlist_dir_, media_playlist.get()))
-        return false;
+    } else if (!WriteMediaPlaylist(master_playlist_dir_, media_playlist.get())) {
+      return false;
```
Comment (kqyang) [inline packager/hls/base/simple_hls_notifier.cc]: I'd prefer keeping the original style for this one, which is more readable.

**Comment by Korilakkuma** [comment]
@kqyang 

> I don't see a semantics change anywhere. Can you clarify what you mean by "early return"?

Sorry, I mistaken. Correctly, not "early return" but "no else return".

``` C++
if (x) {
  return y;
} else {
  return z;
}
```

->

```C++
if (x) {
  return y;
}

return z;
```

**Comment by kqyang** [comment]
Thanks @Korilakkuma. The change has been merged.

**Pull request merged**